packages/oidc-middleware/package.json

```diff
@@ -35,7 +35,6 @@
   "dependencies": {
     "@okta/configuration-validation": "^0.4.1",
     "body-parser": "^1.18.2",
-    "connect-ensure-login": "^0.1.1",
```

Comment:
Per https://oktawiki.atlassian.net/wiki/spaces/eng/pages/814532965/DevEx+JS+Repo+Notes this should update the version and include a CHANGELOG. 

(Note that sign-in-widget follows different rules than the rest of our JS repos)

Reply:
Updated